### PR TITLE
Add nodeagent yaml for testing purpose

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

File: security/cmd/node_agent_k8s/nodeagent.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    spec:
+      initContainers:
+      - name: flexvol-driver
+        image: quay.io/saurabh/flexvol:latest
+        volumeMounts:
+        - name: flexvol-driver-host
+          mountPath: /host/driver
+      containers:
+      - name: nodeagent
+        image: quay.io/saurabh/nodeagent:933707e
```
Comment (wenchenglu) [inline security/cmd/node_agent_k8s/nodeagent.yaml]: replace saurabh with a more general name like foo?

File: security/cmd/node_agent_k8s/nodeagent.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+spec:
+  template:
+    metadata:
+      labels:
+        app: nodeagent
+        version: v1
+    spec:
+      initContainers:
+      - name: flexvol-driver
+        image: quay.io/saurabh/flexvol:latest
```
Comment (wenchenglu) [inline security/cmd/node_agent_k8s/nodeagent.yaml]: what is "quay"?

File: security/cmd/node_agent_k8s/nodeagent.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
```
Comment (wenchenglu) [inline security/cmd/node_agent_k8s/nodeagent.yaml]: will be nice to have some high-level description for this config on the top of this file

Comment (wenchenglu) [inline security/cmd/node_agent_k8s/nodeagent.yaml]: this is a test file right? nodeagent_test.yaml?

Comment (wattli) [reply security/cmd/node_agent_k8s/nodeagent.yaml]: good point, done.

Comment (wattli) [reply security/cmd/node_agent_k8s/nodeagent.yaml]: sure. done

File: security/cmd/node_agent_k8s/nodeagent.yaml (hunk 1 above)
Comment (wattli) [reply security/cmd/node_agent_k8s/nodeagent.yaml]: done.

File: security/cmd/node_agent_k8s/nodeagent.yaml (hunk 2 above)
Comment (wattli) [reply security/cmd/node_agent_k8s/nodeagent.yaml]: it's some repo like gcr, replaced with gcr.io now

**Comment by wattli** [comment]
My bad, the file is added.

**Pull request merged**